chaosgcp/dns/actions.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+from typing import Any, Dict
+
+from chaoslib.types import Configuration, Secrets
+
+from chaosgcp import client
+
+__all__ = ["update_service"]
+
+
+def update_service(
+    project: str = None,
+    ip_address: str = None,
+    name: str = None,
+    zone_name: str = None,
+    kind: str = "dns#resourceRecordSet",
+    ttl: int = 5,
+    record_type: str = "A",
+    existing_type: str = "A",
```

Comment:
Why are all these optionals? I can understand for some of them. But surely `name`, `zone_name` and `ip_address` should be mandatory?